test/e2e/01-install.sh

```diff
@@ -17,6 +17,8 @@ set -e
 
 export KO_DOCKER_REPO=${KO_DOCKER_REPO:-"kind.local"}
 export KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-"tekton-results"}
+export SA_TOKEN_PATH=${SA_TOKEN_PATH:-"/tmp/tekton-results/tokens"}
+export SSL_CERT_PATH=${SSL_CERT_PATH:="/tmp/tekton-results/ssl"}
```

Comment:
That obviously works, but is there a reason you've used `:=` for the parameter expansion in this case, while `:-` is used in the other cases? The same goes in `e2e.sh`.